spec/03-plugins/14-request-transformer/02-access_spec.lua

```diff
@@ -71,6 +82,26 @@ for _, strategy in helpers.each_strategy() do
         }
       }
 
+      bp.plugins:insert {
+        route = { id = route_ph.id },
+        name     = "request-transformer",
+        config   = {
+          add    = {
+            headers = {"x-added-foo:bar"}
+          }
+        }
+      }
+
+      bp.plugins:insert {
+        route = { id = route_phch.id },
+        name     = "request-transformer",
+        config   = {
+          add    = {
```

Comment:
If this was `replace` instead of `add`, would the plugin still not be able to override the Host header when `preserve_host = true`? It seems to me like it would.

Reply:
haha, I had just added these test cases as I was reorganizing the patch! Check out the latest version :)